packages/auth-providers/supertokens/setup/src/setupHandler.ts

```diff
@@ -18,14 +18,15 @@ export async function handler({ force: forceArg }: Args) {
       "import { authDecoder } from '@redwoodjs/auth-supertokens-api'",
     apiPackages: [
       `@redwoodjs/auth-supertokens-api@${version}`,
-      'supertokens-node',
+      'supertokens-node@^12',
     ],
     webPackages: [
       `@redwoodjs/auth-supertokens-web@${version}`,
-      'supertokens-auth-react',
+      'supertokens-auth-react@^0',
+      'supertokens-web-js@^0',
```

Comment:
This is a peerDependency of `supertokens-auth-react`.

Reply:
I'd consider pinning the minor version too. For 0.x versions you're allowed to introduce breaking changes in minors according to SemVer

Reply:
Good call, will do that

Reply:
Done in https://github.com/redwoodjs/redwood/pull/7476/commits/e12294ecad21db01da954510587c748c7ce71440

Reply:
Actually the caret may still upgrade the minor. Let me reread how those semver operators work

Reply:
Looks like it actually handles pre v1 versions the way we want: https://github.com/npm/node-semver#caret-ranges-123-025-004